ceph/ceph.py

```diff
@@ -1281,6 +1281,7 @@ def __init__(self, **kw):
         if kw.get("ceph_vmnode"):
             self.vm_node = kw["ceph_vmnode"]
             self.osd_scenario = self.vm_node.osd_scenario
+        self.id = kw.get("id", None)
```

Comment:
Let's add a check before creating `Ceph` Instance to ensure duplicate node-id(s), which may cause issues in further execution.

Reply:
This validation we have added even before reaching this stage

Reply:
Thanks